### ISSUE TITLE
Interledger White Paper page has a page not found code

### ISSUE THREAD

**Issue by jessiehennion828** [interledger/interledger-website#16]
https://interledger.org/interledger.pdf

**Comment by mccamon** [comment]
I am verifying with @jessiehennion828 but I think the PDF was on the site prior to DNS switch. I checked on the old repo and it was simply a file at the root. Not sure right approach but we could create a /files/ folder in Cloudcannon and upload it there. I'm just a big fan of keeping root clean. @tbeck thoughts?

**Comment by tbeck** [comment]
Hi Mike,
Yes… I absolutely agree on the “keeping root clean” strategy.
If you upload a file through the CMS it will default to the “uploads” directory. We can change this if you would prefer another default directory name though.

Thanks!
Tom

----------------------------------------------------
Tom Beck | Technical Director | Ocupop
www.ocupop.com
Phone : 604.902.1411
Video : ocu.vc/tom
----------------------------------------------------

**Comment by jessiehennion828** [comment]
Mike-
Assuming it was on the previous site.

**Comment by mccamon** [comment]
Okay I’ll do this when back at desk. 


Mike McCamon | m: +1.913.940.2737 | Sent from iPhone

**Comment by mccamon** [comment]
@tbeck I uploaded through CMS it went to root. But I get 404 if I try interledger.org/interledger.pdf

**Comment by nathanielfostanperry** [comment]
@mccamon I was able to upload to both the root and uploads folder in the CMS on staging. Do either/both of these links work as the intent?

[https://frank-beach.cloudvent.net/uploads/interledger.pdf](https://frank-beach.cloudvent.net/uploads/interledger.pdf)

[https://frank-beach.cloudvent.net/interledger.pdf](https://frank-beach.cloudvent.net/interledger.pdf)

**Comment by tbeck** [comment]
@mccamon - If you are okay with this, I would suggest placing the file in the uploads directory and keep the root folder clean. In the meantime, we can also add a 301 redirect to account for any existing instances of the link to the https://interledger.org/insterledger.pdf 

Let me know if this is okay and we will push as part of our EOD merge.

Thanks!

**Comment by mccamon** [comment]
Approved. It is there and in uploads. 301 is good because it's a legacy link from old site. 301s managed in Cloudflare?

**Comment by tbeck** [comment]
Hi @mccamon - 301 can be managed in Cloudcannon using the 301.txt file. Here is a direct link: https://app.cloudcannon.com/32080/editor#sites/88070/browser/src:/browser/src/301.txt

That being said... it looks like there is some issue with the domain. We have submitted a ticket with Cloudcannon  to see if they have any ideas, but we may also need to look at the Cloudflare settings to see if something is in place that is preventing that url from resolving. We should see if we can get credentials from Briana.

**Comment by mccamon** [comment]
Sounds good


Mike McCamon | m: +1.913.940.2737 | Sent from iPhone

**Comment by mccamon** [comment]
Reporting that neither location is working.

https://interledger.org/src/uploads/interledger.pdf << 404
https://interledger.org/interledger.pdf << 404

BTW @tbeck it looks like you 301 with Cloudcannon https://docs.cloudcannon.com/hosting/general/301-redirects/ But until I get the /uploads/ link to work that still doesn't fully solve this issue

**Comment by tbeck** [comment]
Hi @mccamon 
In this build the "src" acts as the root. So the link would be: https://interledger.org/uploads/interledger.pdf
This appears to be working on this end.

We will continue to troubleshoot the redirect though. The Cloudcannon team thought that there may be a configuration setting in the Cloudflare account , however we will need access from Briana to explore that possibility further.

**Comment by mccamon** [comment]
Yes, @tbeck that link did work yesterday... now https://interledger.org/uploads/interledger.pdf resolves to 404. I would try the 301-redirect feature in Cloudcannon but I don't have a link that works as a patch. I also opened a ticket with Cloudcannon.

Lastly I did look aroudn their doc this morning and I can't get to the Site > Settings - maybe there's setting there that impact this sort of file request. (since all the links of files in uploads are oviscaped like when I right mouse click on Briana's image and the link is not simply /uploads/briana-marbury.jpg)
